_studio/mfx_lib/shared/src/mfx_mpeg2_encode_vaapi.cpp

```diff
@@ -20,7 +20,7 @@
 
 #include "mfx_common.h"
 
-#if defined(MFX_ENABLE_MPEG2_VIDEO_ENCODE_HW) && defined(MFX_VA_LINUX)
+#if defined(MFX_ENABLE_MPEG2_VIDEO_ENCODE) && defined(MFX_VA_LINUX)
```

Comment:
Comment for other reviewers: MFX_ENABLE_MPEG2_VIDEO_ENCODE_HW was used only in 2 places actually. It is simpler to remove it completely rather than conditionally define if MFX_ENABLE_MPEG2_VIDEO_ENCODE is defined.